test/testconstructors.cpp

```diff
@@ -275,8 +275,8 @@ class TestConstructors : public TestFixture {
               "    int i;\n"
               "};");
         ASSERT_EQUALS("[test.cpp:4]: (warning) Member variable 'Fred::i' is not initialized in the constructor.\n"
-                      "[test.cpp:5]: (warning) Member variable 'Fred::i' is not initialized in the constructor.\n"
-                      "[test.cpp:6]: (warning) Member variable 'Fred::i' is not initialized in the constructor.\n", errout.str());
+                      "[test.cpp:5]: (warning) Member variable 'Fred::i' is not assigned in the copy constructor. Should it be copied?\n"
```

Comment:
I am not sure. If it is not initialized then it will be uninitialized. That is what we warn about. So I fear a "Should it be copied?" warning could confuse users so they think we want to enforce that all variables are copied. That is somewhat different.

Reply:
I didn't get what `inconclusive` had to do with it, see the comment on the other PR. But I think we should still refer to the correct type of constructor. For the case above, "[test.cpp:5]: (warning) Member variable 'Fred::i' is not initialized in the copy constructor.\n"

Reply:
> I didn't get what `inconclusive` had to do with it

It seems that we have reused uninitMemberVar for 2 different checks. One check that checks for uninitialized member variables.. that is good.. then we have another check that checks that all members are copied/assigned. 

it was very unfortunate to reuse the uninitMemberVar id for the second check. I feel a bit skeptic about that check, does it really belong in cppcheck? Sure in most cases I do want to copy members but then there are special cases when I don't want to copy members.. I would assume that is why this second check is inconclusive. Maybe it would be better to move it to an addon.

Reply:
Hmm, it could be a cause of nasty bugs for sure.

Reply:
> Hmm, it could be a cause of nasty bugs for sure (not copying members).

well.. if we keep it in cppcheck for now and let it be inconclusive.. I really think it should be separated from uninitMemberVar. I don't know.. how about uncopiedMemberVar.